tests/api/test_overdrive.py

```diff
@@ -2233,6 +2233,15 @@ def test_no_drm_fulfillment(self, overdrive_api_fixture: OverdriveAPIFixture):
         assert fulfill.content_link_redirect is True
         assert fulfill.content_link == "https://example.org/epub-redirect"
 
+    def test_bug_fix_pp_1219(self, overdrive_api_fixture: OverdriveAPIFixture):
```

Comment:
Could we give this test method a more descriptive name? I'm guessing something like `test_no_recently_changed_books`.